Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -58,6 +147,23 @@ $checkTestAgentCompatScriptLocationMemberExists  = CmdletHasMember "CheckTestAge
 $checkCustomSlicingEnabledMemberExists  = CmdletHasMember "CustomSlicingEnabled"
 $taskContextMemberExists  = CmdletHasMember "TaskContext"
 
+if($overrideRunParams -and $runSettingsFile -and (Test-Path $runSettingsFile))
+{
+    if (([string]::Compare([io.path]::GetExtension($runSettingsFile), ".testsettings", $True) -eq 0)) {
+        $xml = [xml](Get-Content $runSettingsFile)
+        $hasOverridenProperties = OverrideTestSettingProperties -overridingParameters $overrideRunParams -testSettings $xml
+        if($hasOverridenProperties)
+        {
+            $newTestSettingsFile = [io.path]::Combine($env:TEMP, ([GUID]::NewGuid()).toString() + ".testsettings" )
+            $xml.Save($newTestSettingsFile)
+            Write-Verbose "Task will be using new TestSettings file created after overriding properties : $newTestSettingsFile"
```

Comment:
Localization error for failed test messages

Reply:
added the warning message, but didnt add localization as presently task doesnt have any warnings/messages localized.

---
In reply to: [137521586](https://github.com/Microsoft/vsts-tasks/pull/5246#discussion_r137521586) [](ancestors = 137521586)